packages/eslint-plugin-next/lib/rules/image-component/no-unsized-images.js

```diff
@@ -0,0 +1,36 @@
+const createImageRule = require('../../utils/imageComponentRule.js')
+const NodeAttributes = require('../../utils/nodeAttributes.js')
+
+module.exports = {
+  meta: {
+    messages: {
+      unsizedImages:
+        'For layout stability, the image component should be used ' +
+        'with a height and width property, even if actual image size is determined with CSS.' +
+        'if you cannot provide a correct-ratio height and width, use the "unsized" attribute.' +
```

Comment:
do we know of legit cases for the opt out? is this the right way to opt out vs. via config file?